### PR TITLE
Update Serenity Framework Configurations

### PR DESCRIPTION
**Pull request by turkucuberk**

### CODE REVIEW

**Review by HakanKahramanQA — APPROVED** [review]

File: src/test/java/com/lambdatest/cucumber/ParallelAndroidTest1.java [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 import com.lambdatest.LambdaTestSerenityTest;
-import cucumber.api.CucumberOptions;
+import io.cucumber.junit.CucumberOptions;
 import net.serenitybdd.cucumber.CucumberWithSerenity;
 import org.junit.runner.RunWith;
 
-@RunWith(CucumberWithSerenity.class)
-@CucumberOptions(features = "src/test/resources/features/single.feature")
-public class ParallelAndroidTest1 extends LambdaTestSerenityTest {
+@RunWith(CucumberWithSerenity.class) @CucumberOptions(features = "src/test/resources/features/single.feature") public class ParallelAndroidTest1
```
Comment (sururocks) [inline src/test/java/com/lambdatest/cucumber/ParallelAndroidTest1.java]: why you want to run just specific feature file.

File: src/test/java/com/lambdatest/cucumber/ParallelAndroidTest2.java [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 import com.lambdatest.LambdaTestSerenityTest;
-import cucumber.api.CucumberOptions;
+import io.cucumber.junit.CucumberOptions;
 import net.serenitybdd.cucumber.CucumberWithSerenity;
 import org.junit.runner.RunWith;
 
-@RunWith(CucumberWithSerenity.class)
-@CucumberOptions(features = "src/test/resources/features/single.feature")
-public class ParallelAndroidTest2 extends LambdaTestSerenityTest {
+@RunWith(CucumberWithSerenity.class) @CucumberOptions(features = "src/test/resources/features/single.feature") public class ParallelAndroidTest2
```
Comment (sururocks) [inline src/test/java/com/lambdatest/cucumber/ParallelAndroidTest2.java]: why running is use for specific file only.

File: src/test/java/com/lambdatest/cucumber/steps/TodoAppSteps.java [hunk 3]
```diff
@@ -1,58 +1,50 @@
...
-		Thread.sleep(3000);	
+		Thread.sleep(3000);
 	}
 
 	@Then("^click on geolocation element$")
 	public void click_geolocation()throws InterruptedException {
 		todo.clickOnGeolocation();
 		System.out.println("Clicked geolocation button");
-		Thread.sleep(3000);		
+		Thread.sleep(3000);
```
Comment (sururocks) [inline src/test/java/com/lambdatest/cucumber/steps/TodoAppSteps.java:48]: is thread. sleep needed essentially ?

**Review by sururocks — CHANGES_REQUESTED** [review]
change requested

File: src/test/java/com/lambdatest/cucumber/ParallelAndroidTest1.java (hunk 1 above)
Comment (turkucuberk) [reply src/test/java/com/lambdatest/cucumber/ParallelAndroidTest1.java]: updated

File: src/test/java/com/lambdatest/cucumber/ParallelAndroidTest2.java (hunk 2 above)
Comment (turkucuberk) [reply src/test/java/com/lambdatest/cucumber/ParallelAndroidTest2.java]: updated

**Review by sururocks — APPROVED** [review]

**Pull request merged**